OPSC-17905 OpsCenter Java driver revert announcement (#275)
datastax/release-notes: OpsCenter_6.8_Release_Notes.md

# Release notes for OpsCenter

Due to numerous errors and customers reporting performance problems with the versions of OpsCenter released with the updated 
java driver, we are going to release OpsCenter 6.8.51 with the driver update reverted as soon as possible. We will be removing 
the OpsCenter versions with the driver issue and other significant issues from FixCentral. Those are 6.8.47, 6.8.48, 6.8.49, 
and 6.8.50. We recommend customers encountering issues go back to 6.8.46 until a fixed version of OpsCenter is released. All 

> updates since 6.8.46 that are not related to the driver upgrade will remain in place.

Reviewer: Suggested change
      
    
    
      
          
            
            Due to numerous errors and customers reporting performance problems with the versions of OpsCenter released with the updated 
          
          
            
            java driver, we are going to release OpsCenter 6.8.51 with the driver update reverted as soon as possible. We will be removing 
          
          
            
            the OpsCenter versions with the driver issue and other significant issues from FixCentral. Those are 6.8.47, 6.8.48, 6.8.49, 
          
          
            
            and 6.8.50. We recommend customers encountering issues go back to 6.8.46 until a fixed version of OpsCenter is released. All 
          
          
            
            updates since 6.8.46 that are not related to the driver upgrade will remain in place.
          
          
            
            Released OpsCenter versions 6.8.48, 6.8.49, and 6.8.50 contain an updated Java driver that resulted in numerous errors and performance problems. A follow-on release, OpsCenter version 6.8.51, will revert the driver update.  Fix Central will no longer offer the prior release versions affected by this Java driver update, or version 6.8.47, pulled from Fix Central and fixed in version 6.8.48.  OpsCenter version 6.8.46 and earlier versions remain unaffected and are still available. All updates since version 6.8.46 that are not related to the driver upgrade remain in place.
          
          
            
            We recommend customers encountering issues temporarily revert to version 6.8.46 until the advent of version 6.8.51.

> The OpsCenter team is making significant improvements to the QA process to avoid mistakes like this in the future.

Reviewer: ```suggestion
Significant QA process improvements are in process to avoid similar future issues.
```
